--- a/core/lib/scmi-server/conf-optee-stm32mp2.mk
+++ b/core/lib/scmi-server/conf-optee-stm32mp2.mk
@@ -19,7 +19,5 @@
 
-ifeq ($(call cfg-one-enabled, CFG_STM32MP23 CFG_STM32MP25),y)
-$(call force,CFG_SCPFW_MOD_POWER_DOMAIN,y)
-$(call force,CFG_SCPFW_MOD_SCMI_POWER_DOMAIN,y)
-$(call force,CFG_SCPFW_MOD_STM32_PD,y)
-endif
+$(call force,CFG_SCPFW_MOD_POWER_DOMAIN,n)
+$(call force,CFG_SCPFW_MOD_SCMI_POWER_DOMAIN,n)
+$(call force,CFG_SCPFW_MOD_STM32_PD,n)
 
